lib/collections/schemas/helpers.js

```diff
@@ -99,3 +99,67 @@ export function shopDefaultCountry() {
     return this.value;
   }
 }
+
+export function shopDefaultParcelWeight() {
+  // we should always have a shopId
+  if (this.isSet && Meteor.isServer) {
+    return this.value;
+  } else if (Meteor.isServer && !this.isUpdate || Meteor.isClient && this.isInsert) {
+    const shop = Shops.findOne({
+      _id: Reaction.getShopId()
+    });
+    if (shop && shop.defaultParcelSize) {
+      return shop.defaultParcelSize.weight;
+    }
+    return this.value;
+  }
+  return this.unset();
+}
+
+export function shopDefaultParcelLength() {
```

Comment:
Rather than exporting/importing these all individually, it might be better to create a namespace and just export/import that